articles/libraries/machine-learning/glossary-qml.md

```diff
@@ -7,36 +7,37 @@ ms.topic: article
 uid: microsoft.quantum.libraries.machine-learning.training
 ---
 
-# Classifier Training Process and Results
+# Quantum Machine Learning glossary
 
 Training of a circuit-centric quantum classifier is a process with many moving parts that require the same (or slightly larger) amount of
-calibration by trial and error as training of traditional classifiers. Here we define the main concepts and ingredients of this training process. 
+calibration by trial and error as training of traditional classifiers. Here we define the main concepts and ingredients of this training process.
 
 ## Training/testing schedules
+
 In the context of classifier training a *schedule* describes a subset of data samples in an overall training or testing set. A schedule is usually defined as a
 collection of sample indices.
 
 ## Parameter/bias scores
+
 Given a candidate parameter vector and a classifier bias, their *validation score* is measured relative to a chosen validation schedule S and is expressed by a number of misclassifications
  counted over all the samples in the schedule S.
 
-
 ## Hyperparameters
+
 The model training process is governed by certain pre-set values called *hyperparameters*:
```

Comment:
Given this structure, it might help to link to the API reference for TrainingOptions and mention the use of the `w/` pattern to define a kind of [optional argument](https://quantumcomputing.stackexchange.com/a/9538/1978).